### DISCUSSION TITLE
Add jsonapi-serializer to benchmarks?

### DISCUSSION THREAD

**Discussion by kapso** [okuramasafumi/alba#329] [General]
Are there any benchmarks comparing it with https://github.com/jsonapi-serializer/jsonapi-serializer

https://medium.com/@raj.b.stretz/active-model-serializer-vs-fast-json-api-serializer-8338b939f01f

**Comment by okuramasafumi** [comment]
It was there before, but removed by [this commit](https://github.com/okuramasafumi/alba/commit/72e42e072e9cea9be77735c711f169ddbb2f978c).

**Reply by kapso** [reply]
@okuramasafumi curious why was it removed? Any plans of adding it back?

**Reply by okuramasafumi** [reply]
Please read the commit message.

**Comment by kapso** [comment]
@okuramasafumi  I did some quick benchmarks, and looks like `jsonapi-serializer` is 3x faster than Alba.

AccountSerializer does not have any associations, just attributes.

```ruby
# Active record load
a = Account.first

# Uses jsonapi-serializer
irb(main):250:0> puts Benchmark.measure { 100_000.times { App::V1::AccountSerializer.new(a).to_json } }
  2.455879   0.028570   2.484449 (  2.484457)

# Uses Alba
irb(main):253:0> puts Benchmark.measure { 100_000.times { App::V2::AccountSerializer.new(a).to_json } }
  7.511837   0.066197   7.578034 (  7.578014)
```

**Reply by okuramasafumi** [reply]
`jsonapi-serializer` is only for JSON:API and I think it should be faster than Alba, which can generate any format. The comparison cannot be fair because the purpose of them are different.

**Reply by kapso** [reply]
Yea thats a good point, thanks for your response.
